core/libraries/form_sections/strategies/validation/EE_URL_Validation_Strategy.strategy.php

```diff
@@ -36,7 +43,7 @@ public function validate($normalized_value)
         if ($normalized_value) {
             if (filter_var($normalized_value, FILTER_VALIDATE_URL) === false) {
                 throw new EE_Validation_Error($this->get_validation_error_message(), 'invalid_url');
-            } else {
+            } elseif (apply_filters('FHEE__EE_URL_Validation_Strategy__validate__check_remote_file_exists', $this->check_file_exists, $this->_input)) {
```

Comment:
Just observing that this changes the default behaviour from validating full urls to _not_ validating full urls.  I realize this is intentional, but it should be noted on release so any developers/site owners _expecting_ the previous default and wanting to preserve it are made aware of how to do so.  So maybe a blog entry in developer.eventespresso.com as well?